CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Search/Filter pane is not in a collapsed state. Refs STSMACOM-677.
 * Mark `initialValues` as immutable in `<LocationForm>`. Fixes STSMACOM-679.
 * Fix `<SearchAndSort>` query doesn't reset when clicking back in browser. Fixes STSMACOM-676.
+* Query is copied to search field when navigating back and forth. Fixes STSMACOM-681.
```

Comment:
Is the query supposed to be copied into the search field when navigating and this PR adds it, or is that behavior wrong and this PR changes it? I cannot tell from reading this description. Use declarative sentences in CHANGELOGs and commit messages like, "Copy query into search field when..." or "Omit query from search field when..."

Reply:
Great! I will fix that and use it in the future. I just took the name of the ticket.